src/database/schema/ChannelListSchema.ts

```diff
@@ -86,6 +89,7 @@ class ChannelList implements BaseDbSchema {
     this.anon_user_info_color_name = anon_user_info_color_name;
     this.anon_user_info_emoji_name = anon_user_info_emoji_name;
     this.anon_user_info_emoji_code = anon_user_info_emoji_code;
+    this.memberUsers = memberUsers;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `memberUsers` property is being set in the constructor, which is expected behavior.
```diff
+    this.memberUsers = memberUsers;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->